jasmine/spec/inverted-index-spec.js

```diff
@@ -0,0 +1,79 @@
+// Instanciate new Index
+var index = new Index();
+
+// Run createIndex method and make sure it completes before any test is run
+beforeEach(function(done) {
+    index.createIndex('/jasmine/spec/test.json').then(function() {
+        done();
+
```

Comment:
**Styling - Critical**

You have many unnecessary blank lines. You do not need a blank line here as you do on line 78.
It's advisable to use [AirBnB](https://github.com/airbnb/javascript) JS style guide